Tests/Helper/Mocks/MockBackendService.swift

```diff
@@ -6,6 +6,7 @@
 //  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
 //
 
+import BrightFutures
```

Comment:
Is this import necessary? If not, let's remove it.

Reply:
Removed.